javascript/app.js

```diff
@@ -25,6 +25,14 @@ FinancialFreedom.config(['$routeProvider', '$locationProvider', function($routeP
         templateUrl: 'partials/about.html',
         controller: ''
     })
+    .when('/privacy', {
+        templateUrl: 'partials/legal/privacy_policy.html',
+        controller: ''
+    })
+    .when('/terms-of-service', {
+        templateUrl: 'partials/legal/terms_of_service.html',
+        controller: ''

```

Comment:
what happens when there's no controller? can it be left out entirely if it's empty?